core/templates/dev/head/components/state-directives/rule-editor/rule-editor.directive.ts

```diff
@@ -29,6 +29,7 @@ require(
   'state-property.service.ts');
 require('services/ValidatorsService.ts');
 
+var DEFAULT_OBJECT_VALUES = require('objects/object_defaults.json');
```

Comment:
:open_mouth: It is actually possible to require JSON files directly?

Reply:
Yeah.
I just discovered that :)